attestation-service/docs/grpc-as.md

```diff
@@ -2,15 +2,73 @@
 
 `grpc-as` is an Attestation Service application based on gRPC protocol.
 
-## API
+Now the following types of evidence are supported:
+- `azsnpvtpm`: Azure SNP vTPM
+- `sev`: AMD SEV (Not implemented)
+- `sgx`: Intel SGX
+- `snp`: AMD SNP
+- `tdx`: Intel TDX
+- `cca`: Arm CCA
+- `sample`: A fake platform. Only for test and sample
+- `csv`: Hygon CSV
+- `aztdxvtpm`: Azure TDX vTPM (Not implemented)
 
-gRPC Attestation Service provides gRPC endpoints which is defined in [protobuf](../protos/attestation.proto).
+## Quick Start
 
-## Usage
+Users can use a [community version of gRPC CoCoAS image](https://github.com/confidential-containers/trustee/pkgs/container/staged-images%2Fcoco-as-grpc) to verify attestation reports.
 
-Here are the steps of building and running gRPC Attestation Service:
+```shell
+# run gRPC CoCoAS server locally
+docker run -d \
+  -v /etc/sgx_default_qcnl.conf:/etc/sgx_default_qcnl.conf \ # this qcnl config is used when verifying SGX/TDX quotes
+  -p 50004:50004 \
+  ghcr.io/confidential-containers/staged-images/coco-as-grpc:latest
+```
+
+The `sgx_default_qcnl.conf` is configure the PCCS of Intel platforms. An example file is given [here](../../kbs/config/sgx_default_qcnl.conf).
+You can set `pccs_url` to an available PCCS. PCCS are usually supported by cloud providers, you can find the steps to configure `/etc/sgx_default_qcnl.conf` for
```

Comment:
@surajssd's question above has another meaning: users don't know where the file is coming and most likely it's not available on `/etc`.  Perhaps make that `docker run` to bind mount what's in this repo?